content/case_histories/hoop_region_norway/processinginterpretation.rst

```diff
@@ -0,0 +1,210 @@
+
+.. (Alvarez et al. 2015)
+
+
+Multiple elastic attributes such as P-wave impedance :math:`Ip`, S-wave impedance :math:`Is`, P-to-S velocity ratio (:math:`Vp ∕ Vs`), the product of density and Lamé’s parameters :math:`\lambda \rho` and :math:`\mu \rho` (Goodway et al., 1997), Poisson’s ratio :math:`\sigma`, the product of density by bulk modulus :math:`\kappa \rho`, the product of density and dynamic Young’s modulus :math:`E \rho`, Poisson dampening factor (:math:`PDF`) (:cite:`Mazumdar2007`), etc., can be used in the MARS assessment. For this case study, for each target petrophysical property, MARS was run for a 2D combination of the 64 elastic attributes shown in :numref:`fig_table_seismic_attributes` , which can be derived from :math:`Ip` and :math:`Is`, resulting in the evaluation of 2016 independent bi-dimensional spaces. In this table, each number represents a single attribute, which is obtained after applying the mathematical operation shown in the leftmost column to the elastic attribute shown in the uppermost row. For example, the number 21 represents the attribute. The purpose of applying a mathematical operation (such as square root, power, inverse, logarithm, etc.) to attributes is to be able to model physical phenomena that exhibit nonlinear behavior. This is a mathematical strategy to linearize potential nonlinear relationship between the elastic attributes and the petrophysical properties, used with the goal of improving the correlation between the attribute t and the target petrophysical property.
+
+
+.. figure:: images/table_seismic_attributes.png
+    :align: center
+    :figwidth: 100%
+    :name: fig_table_seismic_attributes
+
+    Matrix of attributes used in MARS. Each number represents a single attribute, which is obtained after applying the mathematical operation show in the leftmost column to the uppermost row. For example, the number 21 represents the attribute.
+
+
+The MARS approach was applied in two different depth windows, given the different rock physics relationships between the elastic attributes and the petrophysical properties in the Stø and Nordmela Fms. (see :numref:`fig_poisson_acoustic_impedance`). The first window comprises the Fuglen and Stø Formations and the second window the Nordmela Formation. The rock properties that were estimated using the MARS approach were total porosity (:math:`\phi_T`), volume of clay (Vclay) and the hybrid petrophysical property water saturation plus volume of clay (Sw+Vclay).
```

Comment:
Sw is a variable in Simandoux's equation but the other two weren't firmly defined as such. I can put them into math script if you think it would look better.